packages/@react-spectrum/toast/src/ToastContainer.tsx

```diff
@@ -48,8 +49,25 @@ export function clearToastQueue() {
   globalToastQueue = null;
 }
 
-let toastProviders = new Set();
+
+let store = new Set();
+let toastProviders = {
+  add: (ref) => {
+    store.add(ref);
+    emitChange();
```

Comment:
Why do we `emitChange()` on an `add` now? Previous code only did this on a `delete`.